### ISSUE TITLE
Residents start page: Oxfordshire - variable maximum permits

### ISSUE THREAD

**Issue by AbiBarkerOxfordshire** [alphagov/verify-local-patterns#549]
Oxfordshire either has no limits on the number of permits that can be issued or a maximum of 2 permits. As we are unable to check at the time whether someone is eligible due to our lack of API is it possible for any reference to the number of permits being removed and to be able to add something that says 'We will consider your application and let you know the outcome' or something similar?

We do not offer any free permits for any applications so would also like to remove any reference to this.

**Comment by EUzkuraityte** [comment]
@AbiBarkerOxfordshire yes, that sounds reasonable assuming that if such APIs would become available to you in the future, you would implement the functionality. 

and yes, you can tailor the application to suit your business rules re: free permits, etc. 

I only ask that once you settle on the copy you want to be displayed, could you tag in @lizziebruce so that if she has some free time she could check the advice to users to make sure it's user friendly, etc.

**Comment by AbiBarkerOxfordshire** [comment]
@lizziebruce Hi Lizzie - our page currently says:

'As a resident of Oxfordshire you can buy a permit to park near your home 24 hours a day, 7 days a week.

You can apply for a maximum 1 permit per household' - we have a rule that it is 1 permit per person, per household,  but the maximum per household caries depending on the area and CPZ in place - 
the applicant should already have checked any restrictions on the previous page so can I change this to: 

'As a resident of Oxford you can apply for a permit to park near your home 24 hours a day, 7 days a week.

You can apply for a maximum of 1 permit per person, per household.' ?

**Comment by lizziebruce** [comment]
@AbiBarkerOxfordshire Hi Abi, I'm a little unclear from the copy if "1 per person, per household" means each person in the household can get one permit each, or only one permit can be issued to a household?

If it's 1 per person, I'd suggest:
"You can apply for a maximum of 1 permit per person."

If it's 1 per household, I'd suggest:
"You can apply for a maximum of 1 permit per household."
